apps/admin-portal/src/components/identity-providers/wizard/steps/wizard-summary.tsx

```diff
@@ -82,26 +82,28 @@ export const WizardSummary: FunctionComponent<WizardSummaryProps> = (
         });
     };
 
-    return (
-        <Grid className="wizard-summary">
-            <Grid.Row>
-                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 } textAlign="center">
-                    <div className="general-details">
-                        <AppAvatar
-                            name={ identityProvider?.name }
-                            image={ identityProvider?.image }
-                            size="tiny"
-                        />
-                        {identityProvider?.name && (
-                            <Heading size="small" className="name">{identityProvider.name}</Heading>
-                        )}
-                        {identityProvider?.description && (
-                            <div className="description">{identityProvider.description}</div>
-                        )}
-                    </div>
-                </Grid.Column>
-            </Grid.Row>
+    const getGeneralDetailsComponent = () => {
+        return <Grid.Row>
+            <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 } textAlign="center">
+                <div className="general-details">
+                    <AppAvatar
+                        name={ identityProvider?.name }
+                        image={ identityProvider?.image }
+                        size="tiny"
+                    />
+                    {identityProvider?.name && (
+                        <Heading size="small" className="name">{identityProvider.name}</Heading>
+                    )}
+                    {identityProvider?.description && (
+                        <div className="description">{identityProvider.description}</div>
+                    )}
+                </div>
+            </Grid.Column>
+        </Grid.Row>;
+    };
 
+    const getAuthenticatorComponent = () => {
+        return <>
             <Divider horizontal>Authenticator details</Divider>
 
             {authenticatorSummary && (
```

Comment:
shall we get the circuit breaker to the next line?

Reply:
Fixed with aaccd78903a82df120e76ba561155546a19c6204.